include/nbl/video/SPhysicalDeviceLimits.h

```diff
@@ -0,0 +1,497 @@
+    //uint32_t              maxInlineUniformBlockSize;
+    //uint32_t              maxPerStageDescriptorInlineUniformBlocks;
+    //uint32_t              maxPerStageDescriptorUpdateAfterBindInlineUniformBlocks;
+    //uint32_t              maxDescriptorSetInlineUniformBlocks;
+    //uint32_t              maxDescriptorSetUpdateAfterBindInlineUniformBlocks;
+    //uint32_t              maxInlineUniformTotalSize;
+    //VkBool32              integerDotProduct8BitUnsignedAccelerated;
+    //VkBool32              integerDotProduct8BitSignedAccelerated;
+    //VkBool32              integerDotProduct8BitMixedSignednessAccelerated;
+    //VkBool32              integerDotProduct4x8BitPackedUnsignedAccelerated;
+    //VkBool32              integerDotProduct4x8BitPackedSignedAccelerated;
+    //VkBool32              integerDotProduct4x8BitPackedMixedSignednessAccelerated;
+    //VkBool32              integerDotProduct16BitUnsignedAccelerated;
+    //VkBool32              integerDotProduct16BitSignedAccelerated;
+    //VkBool32              integerDotProduct16BitMixedSignednessAccelerated;
+    //VkBool32              integerDotProduct32BitUnsignedAccelerated;
+    //VkBool32              integerDotProduct32BitSignedAccelerated;
+    //VkBool32              integerDotProduct32BitMixedSignednessAccelerated;
+    //VkBool32              integerDotProduct64BitUnsignedAccelerated;
+    //VkBool32              integerDotProduct64BitSignedAccelerated;
+    //VkBool32              integerDotProduct64BitMixedSignednessAccelerated;
+    //VkBool32              integerDotProductAccumulatingSaturating8BitUnsignedAccelerated;
+    //VkBool32              integerDotProductAccumulatingSaturating8BitSignedAccelerated;
+    //VkBool32              integerDotProductAccumulatingSaturating8BitMixedSignednessAccelerated;
+    //VkBool32              integerDotProductAccumulatingSaturating4x8BitPackedUnsignedAccelerated;
+    //VkBool32              integerDotProductAccumulatingSaturating4x8BitPackedSignedAccelerated;
+    //VkBool32              integerDotProductAccumulatingSaturating4x8BitPackedMixedSignednessAccelerated;
+    //VkBool32              integerDotProductAccumulatingSaturating16BitUnsignedAccelerated;
+    //VkBool32              integerDotProductAccumulatingSaturating16BitSignedAccelerated;
+    //VkBool32              integerDotProductAccumulatingSaturating16BitMixedSignednessAccelerated;
+    //VkBool32              integerDotProductAccumulatingSaturating32BitUnsignedAccelerated;
+    //VkBool32              integerDotProductAccumulatingSaturating32BitSignedAccelerated;
+    //VkBool32              integerDotProductAccumulatingSaturating32BitMixedSignednessAccelerated;
+    //VkBool32              integerDotProductAccumulatingSaturating64BitUnsignedAccelerated;
+    //VkBool32              integerDotProductAccumulatingSaturating64BitSignedAccelerated;
+    //VkBool32              integerDotProductAccumulatingSaturating64BitMixedSignednessAccelerated;
```

Comment:
mostly `VK_KHR_shader_integer_dot_product` untill here, I don't think we want these?

Reply:
but if we did, they'd all be false for OpenGL & ES